source/development/packaging/generate-aix-package.rst

```diff
@@ -18,7 +18,7 @@ Download our wazuh-packages repository from GitHub and go to the aix directory.
 
 .. code-block:: console
 
- $ curl -L https://github.com/wazuh/wazuh-packages/tarball/master | tar zx
+ $ curl -L https://github.com/wazuh/wazuh-packages/tarball/4.0 | tar zx
```

Comment:
Add a variable `WAZUH_PACKAGES_BRANCH` to `conf.py` and replace all harcoded 4.0 values

Reply:
Fixed in https://github.com/wazuh/wazuh-documentation/pull/3864/commits/27a954b305156beaa2b68798932d0dc9cc7ea3ec